client/src/App.tsx

```diff
@@ -29,30 +25,37 @@ function App(): JSX.Element {
       <ThemeProvider theme={theme}>
         <QueryClientProvider client={queryClient}>
           <ReactQueryDevtools />
-          <UserProvider>
-            <ModalProvider>
-              <BrowserRouter
-                getUserConfirmation={() => {
-                  // intentionally left empty callback to block the default browser prompt.
-                }}
-              >
-                <Grid
-                  className="App"
-                  display="flex"
-                  flexDirection="column"
-                  height="100vh"
-                  width="100vw"
-                  justifyContent="space-between"
+          <ErrorBoundary
+            FallbackComponent={ErrorFallback}
+            onReset={() => {
+              /* reset logic here */
+            }}
+          >
+            <UserProvider>
+              <ModalProvider>
+                <BrowserRouter
+                  getUserConfirmation={() => {
+                    // intentionally left empty callback to block the default browser prompt.
+                  }}
                 >
-                  {/* move modal to below footer after troubleshooting */}
-                  <Modal />
-                  <Header />
-                  <Main />
-                  <Footer />
-                </Grid>
-              </BrowserRouter>
-            </ModalProvider>
-          </UserProvider>
+                  <Grid
+                    className="App"
+                    display="flex"
+                    flexDirection="column"
+                    height="100vh"
+                    width="100vw"
+                    justifyContent="space-between"
+                  >
+                    {/* move modal to below footer after troubleshooting */}
```

Comment:
I'm assuming this comment was added before you started working on this, but do you know what this comment is about?

Reply:
Truly not sure, might need to go a few commits back to see the original placer of the said Modal position. I created the Modal and modal provider originally but do not remember adding the comment. I've since moved it to below footer for order but to my best understanding this doesn't directly impact its appearance or position on the screen, especially since it's styled with absolute positioning